### ISSUE TITLE
OS freezes after being switched into sleep mode.

### ISSUE THREAD

**Issue by ch-love** [Mistium/Origin-OS#171]
The title speaks for it self, I put the OS into sleep mode and it suddenly froze not allowing me to interact with it in anyway.

**Comment by Mistium** [comment]
very odd, it might be due to the fps limit changing to 10 , did you try holding down left click?

**Comment by ch-love** [comment]
Yes I did, it still didn't let me interact with it without having to refresh the page.

**Comment by Mistium** [comment]
i will try to replicate the problem. if you get the same thing again, check the browser console for errors since they might help me fix it

**Comment by ch-love** [comment]
Alright. thank you in advanced!

**Comment by Mistium** [comment]
out of interest, did the time display continue?

**Comment by ch-love** [comment]
Yes it did.

**Comment by Mistium** [comment]
pretty sure this is user error, ill improve how responsive the sleep mode is to hopefully avoid it again
